modules/op-adding-triggers.adoc

```diff
@@ -3,9 +3,9 @@
 // *openshift_pipelines/creating-applications-with-cicd-pipelines.adoc
 
 [id="adding-triggers_{context}"]
-= Adding Triggers to a Pipeline
+= Adding triggers to a pipeline
 
-Triggers enable pipelines to respond to external GitHub events, such as push events and pull requests. After you assemble and start a Pipeline for the application, add the `TriggerBinding`, `TriggerTemplate`, `Trigger`, and `EventListener` resources to capture the GitHub events.
+Triggers enable pipelines to respond to external GitHub events, such as push events and pull requests. After you assemble and start a pipeline for the application, add the `TriggerBinding`, `TriggerTemplate`, `Trigger`, and `EventListener` resources to capture the GitHub events.
 
```

Comment:
small indentation issue in this file.. line number 120
```
apiVersion: triggers.tekton.dev/v1alpha1
kind: Trigger
metadata:
  name: vote-trigger
spec:
  serviceAccountName: pipeline
  bindings:
    - ref: vote-app
  template:
     ref: vote-app
```

Reply:
This is the last tiny issue